Banyan/src/clusters.jl

```diff
@@ -183,33 +180,41 @@ get_running_clusters(args...; kwargs...) = filter(entry -> entry[2].status == :r
 function get_cluster_status(name::String=get_cluster_name(), kwargs...)
     c = get_clusters(; kwargs...)[name]
     if c.status == :failed
-        @info c.status_explanation
+        @error c.status_explanation
```

Comment:
I don't think `get_cluster_status` should throw an error if the cluster is failed...because the purpose of this function is to just get the status.

Reply:
This is just an `@error` - it will just print something out.